cv.html

```diff
@@ -242,22 +242,31 @@
                     Freelancer/subcontractor for <a href="https://makaluinc.com/">Makalu Interactive</a>
                     <div class="date-and-title">
                       <div class="date">
-                          2017.09-
+                          2017.09-2025
                       </div>
                     </div>
                     <br/>
                 </div>
                 <div id="body">
-                    <p>I work as a Ruby on Rails/React Native developer on various projects</p>
+                    <p>I work as a React Native/Ruby on Rails developer on various projects with a focus on React Native development</p>
                 </div>
                 <div class="verbose">
                   <ul class="noindent">
-                    <li><a href="https://mymalls.com/">mymalls.com</a> This is an ongoing Rails/React Native project of maintaining the public web app and internal web and mobile app</li>
+                    <li><a href="https://zabit.com/">Zabit</a> React Native habit tracker app (2024.08-2025.08)</li>
+                      <ul>
+                        <li>React Native expert: worked on their React Native habit tracker app, implementing features and improving performance</li>
```

Comment:
So my Makalu ended in 2022 April and after until 2024 August I worked directly to my malls.com.